python/python/bystro/ancestry/inference.py

```diff
@@ -229,40 +227,79 @@ def infer_ancestry(ancestry_models: AncestryModels, genotypes: Dataset) -> Ances
 
     logger.info("Completed ancestry model selection in %f seconds", timer.elapsed_time)
 
-    with Timer() as timer:
-        # Directly concatenate the batches using a generator expression
-        genotypes_df = pd.concat(_generate_batches(scanner))
+    samples = [name for name in genotypes.schema.names if name != "locus"]
 
-    logger.info(
-        "Memory usage after dosage matrix filtering: %s (MB)",
-        psutil.Process(os.getpid()).memory_info().rss / 1024**2,
-    )
+    # Take chunks of up to 500 samples
```

Comment:
Lmk if I'm reading this wrong, but isn't the default chunk size 200 samples on line 34, or is it that the max 500 is a suggestion here?